src/assets/scripts/client/aircraft/FlightManagementSystem/LegModel.js

```diff
@@ -365,7 +365,10 @@ export default class LegModel {
      * @return {array<WaypointModel>}
      */
     _buildWaypointForHoldingPatternAtPosition(holdWaypointProps) {
-        return [new WaypointModel(holdWaypointProps)];
+        // TODO: Why do we return an array of a single waypoint? Is it necessary?
+        const waypointModels = [new WaypointModel(holdWaypointProps)];
+
+        return waypointModels;
```

Comment:
This should actually be:
```javascript
const waypointModel = new WaypointModel(holdWaypointProps);

return [waypointModel];
```

I should add a comment on the why, I pretty sure I have others that explain the why.  But, _the why_: We are returning an array even if it's only a single waypoint because the `LegModel`, and subsequently the `#waypointCollection`, operate on arrays.  this gives us the ability to swap waypoints in an out very easily.  when there is a single waypoint, we still need it to be in an array so that if we ever need to add another waypoint to a leg, we can.  This also keeps our data types consistent.  I will admit, this does feel a little clumsy when we are dealing with a single waypoint in the collection, but thats why we have comments.

Once this gets merged into 139 I will add a comment to that effect.